config/config.default.php

```diff
@@ -365,6 +365,15 @@
     // Enable (food) vouchers
     'enable_voucher'          => (bool) env('ENABLE_VOUCHER', true),
 
+    // Food Voucher Api
+    'food_voucher_api' => [
+        'info_url' => (string) env('FOOD_VOUCHER_API_INFO_URL'),
+        'post_url' => (string) env('FOOD_VOUCHER_API_POST_URL'),
+        'auth_token' => (string) env('FOOD_VOUCHER_API_AUTH_TOKEN'),
+        'default_email' => (string) env('FOOD_VOUCHER_API_DEFAULT_EMAIL'),
+        'redirect_url' => (string) env('FOOD_VOUCHER_API_REDIRECT_URL'),
```

Comment:
```suggestion
        'claim_redirect_url' => (string) env('FOOD_VOUCHER_API_CLAIM_REDIRECT_URL'),
```

or something in that direction would make the meaning clearer?